[BUG] flake8 rejecting commits due to existing code

Despite making no changes to the base.py file, flake8 is failing on that file and not allowing me to commit changes to another file.

```
flake8...................................................................Failed                                                                                                                
- hook id: flake8                                                                                                                                                                              
- exit code: 1                                                                                                                                                                                 
                                                                                                                                                                                               
minihack/base.py:364:25: B907 'key' is manually surrounded by quotes, consider using the `!r` conversion flag.                                                                                 
                        f'Observation key "{key}" is not supported'                                                                                                                            
                        ^                                                                                                                                                                      
minihack/base.py:451:13: B904 Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling.  See https
://docs.python.org/3/tutorial/errors.html#exception-chaining for details.                                                                                                                      
            raise RuntimeError(f"Couldn't patch the nhdat file.\n{e}")                                                                                                                         
            ^                                                                                                                                                                                  
minihack/base.py:523:29: B905 `zip()` without an explicit `strict=` parameter.                                                                                                                 
        for letter, line in zip(inv_letters, inv_strs):                                                                                                                                        
                            ^                                                                                                                                                                  
                                                                                                                                                                                               
[WARNING] Stashed changes conflicted with hook auto-fixes... Rolling back fixes... 
```

Despite trying to add those error codes to the `.flake8` ignore, they are still not being ignored. Not sure how that all works to be honest.

--- a/.flake8
+++ b/.flake8
@@ -1,5 +1,5 @@
 [flake8]
-ignore = E203, E266, E501, W503, F403, F401, B008
+ignore = E203, E266, E501, W503, F403, F401, B008, B905
 max-line-length = 79
 max-complexity = 18
 select = B,C,E,F,W,T4,B9

--- a/minihack/base.py
+++ b/minihack/base.py
@@ -518,7 +518,7 @@ class MiniHack(NetHackStaircase):
         inv_strs = self.last_observation[inv_strs_index]
         inv_letters = self.last_observation[inv_letters_index]
 
-        for letter, line in zip(inv_letters, inv_strs, strict=True):
+        for letter, line in zip(inv_letters, inv_strs):
             if np.all(line == 0):
                 break
             if name in line.tobytes().decode("utf-8"):

--- a/minihack/base.py
+++ b/minihack/base.py
@@ -3,11 +3,11 @@
 import os
 import subprocess
 import random
-from typing import Tuple
-
 import gym
 import numpy as np
 import pkg_resources
+from typing import Tuple
+
 from nle import _pynethack, nethack
 from nle.nethack.nethack import SCREEN_DESCRIPTIONS_SHAPE, OBSERVATION_DESC
 from nle.env.base import FULL_ACTIONS, NLE_SPACE_ITEMS
@@ -360,9 +360,7 @@ class MiniHack(NetHackStaircase):
                         dtype=np.uint8,
                     )
                 else:
-                    raise ValueError(
-                        f'Observation key "{key}" is not supported'
-                    )
+                    raise ValueError(f"Observation key {key} is not supported")
 
         return obs_space_dict
 
@@ -448,7 +446,7 @@ class MiniHack(NetHackStaircase):
                 ]
             )
         except subprocess.CalledProcessError as e:
-            raise RuntimeError(f"Couldn't patch the nhdat file.\n{e}")
+            raise RuntimeError(f"Couldn't patch the nhdat file.\n{e}") from e
 
     def _get_observation(self, observation):
         # Overrides parent class's method to allow for cropping, fitlering out
@@ -520,7 +518,7 @@ class MiniHack(NetHackStaircase):
         inv_strs = self.last_observation[inv_strs_index]
         inv_letters = self.last_observation[inv_letters_index]
 
-        for letter, line in zip(inv_letters, inv_strs):
+        for letter, line in zip(inv_letters, inv_strs, strict=True):
             if np.all(line == 0):
                 break
             if name in line.tobytes().decode("utf-8"):

